src/main/java/com/box/sdk/MetadataTemplate.java

```diff
@@ -453,6 +458,39 @@ public static void deleteMetadataTemplate(BoxAPIConnection api, String scope, St
                             null, fields);
     }
 
+    /**
+     * Executes a metadata query.
+     *
+     * @param api The API connection to be used
+     * @param from The template used in the query. Must be in the form scope.templateKey
+     * @param query The logical expression of the query
+     * @param queryParameters Required if query present. The arguments for the query
+     * @param ancestorFolderId The folder_id to which to restrain the query
+     * @param indexName The name of the Index to use
+     * @param orderBy The field_key(s) to order on and the corresponding direction(s)
+     * @param limit Max results to return for a single request (0-100 inclusive)
+     * @param marker The marker to use for requesting the next page
+     * @return An iterable of BoxMetadataQueryItem search results
+     * @deprecated use executeMetadataQuery() that returns BoxItem.Info instead.
+     */
+    @Deprecated
+    public static BoxResourceIterable<BoxMetadataQueryItem> executeMetadataQuery(final BoxAPIConnection api,
```

Comment:
At this moment we lack `executeMetadata()` method that returns `BoxResourceIterable<BoxMetadataQueryItem> ` and also takes `fields` as parameter. Without that, I don't think we can receive some fields e.g. `metadata`.